torntools/scripts/js/globalFunctions.js

```diff
@@ -1507,10 +1507,10 @@ const STORAGE = {
 				highlight_bloodbags: "none",
 				show_missing_plushies: false,
 				show_missing_flowers: false,
-				show_book_effects: true,
-				show_alcohol_nerve_gains: true,
-				show_candy_happy_gains: true,
-				show_e_can_gains: true,
+				show_book_effects: true, // Debate: false by default
```

Comment:
Even the code is mine and Allo's, this is Mephiles's Idea so I think he should handle it.

Reply:
Book effects are quite useful, can't see why people would want to turn them off. I'd think the majority uses them so leave it on.

Reply:
I was mostly thinking because of mobile, but it seems fine.